Makefile

```diff
@@ -77,6 +77,9 @@ else
 	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm test sh ./tests/Behat/run-tests.sh --profile v2-tests-goutte --tags @v2
 endif
 
+behat-tests-v2-goutte: up-app-integration-tests reset-fixtures disable-debug
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm test sh ./tests/Behat/run-tests-v2.sh
+
```

Comment:
We probably want to give this a different name or drop the old existing version above to save some make confusion

Reply:
Oh yes! Nice spot! I didn't use this make command in the end and had forgot to rename.. made everything called parallel now to be a bit more descriptive especially as we are going to be ditching v2 name when it's the only one at some point :)